src/backend/tests/partaj/core/test_api_referral.py

```diff
@@ -1387,5 +1413,50 @@ def test_unassign_unit_referral_from_invalid_state(self, _):
         referral.refresh_from_db()
         self.assertEqual(referral.units.count(), 2)
         self.assertEqual(
-            models.ReferralActivity.objects.count(), 0,
+            models.ReferralActivity.objects.count(),
+            0,
+        )
+
+    def test_change_urgencylevel(self, _):
```

Comment:
I think we need a few more tests:

- anonymous users cannot do this
- referral user cannot do this
- regular unit member cannot do this
- unit admin can do this
- missing justification
- missing or wrong urgency level id